### PR TITLE
Reduced Maintainer's image load time

### LINKED ISSUE #354: Maintainer image load size can be reduced
**Is your feature request related to a problem? Please describe.**
The maintainer's Image is loaded at the default size **460 x 460** which is not required and is causing a high image load size. Hence, it will take longer time to load the website on devices with a poor network connection.

**Describe the solution you'd like**
The maintainer's image should be loaded at the size of the container by appending **&s=[size of the image required]** to the GitHub avatar URL.

**Additional context**
For @EmmaDawsonDev's GitHub **120 x 120** profile picture, the link will be [**`https://avatars.githubusercontent.com/u/57045550?v=4&s=120`**](https://avatars.githubusercontent.com/u/57045550?v=4&s=120).
For @ctoffanin's GitHub **120 x 120** profile picture, the link will be [**`https://avatars.githubusercontent.com/u/1682188?v=4&s=120`**](https://avatars.githubusercontent.com/u/1682188?v=4&s=120)

**This will reduce the image load size by about 90%. Reference : https://github.com/nyancodeid/quotes/issues/71#issue-1015935234**

### PR DESCRIPTION
**Pull request by SaptarshiSarkar12**
## Describe your changes
Replaced @EmmaDawsonDev's GitHub profile picture link with **https://avatars.githubusercontent.com/u/57045550?v=4&s=150** and @ctoffanin's GitHub profile picture link with **https://avatars.githubusercontent.com/u/1682188?v=4&s=150**. This github profile picture cdn link will provide a **150 x 150** image which does not make any visual difference for the end user, but, it reduces the network payload by about 87%.

## Screenshots - If Any (Optional)
Final Output

## Link to issue

Closes #354 

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [X] I ran the app and tested it locally to verify that it works as expected.
- [ ] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
The resultant file size changes :
| User | Before | After |
|--------|--------|--------|
| @EmmaDawsonDev | 226.7 kB | 31.6 kB |
| @ctoffanin | 21.3 kB | 4 kB |

### CODE REVIEW

**Comment by SaptarshiSarkar12** [comment]
@EmmaDawsonDev @ctoffanin Please review this PR.

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (at-the-vr) [inline .yarnrc.yml:1]: Is this important here to use ? I have seen this for some Migration based chores in GitHub afaik

File: yarn.lock
Comment (at-the-vr) [inline yarn.lock:1]: Not sure if this is required in first place for diagnosing the Issue in hand

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Please revert the commit and then only recommit changes to the relevant file and not the yarn files. 

Also, is there a reason why the whole image url needs changing?

File: data/maintainers.ts
Comment (at-the-vr) [inline data/maintainers.ts:1]: Nice way to use GitHub's CDN. There is a significant reduction in network waterfall, and changing profile pictures will be reflected over here as well 👍

File: .yarnrc.yml (hunk 1 above)
Comment (SaptarshiSarkar12) [reply .yarnrc.yml:1]: Yeah @at-the-vr as the server fails to start. I don't know why.
Here's the error :
```bash
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - SWC minify release candidate enabled. https://nextjs.link/swcmin
warn  - Attempted to load @next/swc-linux-x64-gnu, but an error occurred: libssl.so.1.1: cannot open shared object file: No such file or directory
warn  - Attempted to load @next/swc-linux-x64-gnux32, but it was not installed
warn  - Attempted to load @next/swc-linux-x64-musl, but it was not installed
warn  - Error: EROFS: read-only filesystem, mkdir '/node_modules/next/wasm'
    at makeError$1 (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22607:24)
    at EROFS (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22634:10)
    at ZipFS.mkdirSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24546:13)
    at ZipFS.mkdirpSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22828:16)
    at ZipFS.mkdirSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24544:19)
    at ZipFS.mkdirPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24540:17)
    at /home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25607:26
    at /home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25789:81
    at ZipOpenFS.getZipPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25896:22)
    at ZipOpenFS.makeCallPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25789:23) {
  code: 'EROFS'
}
error - Failed to load SWC binary for linux/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
```

File: data/maintainers.ts
Comment (SaptarshiSarkar12) [reply data/maintainers.ts:1]: Thank you @at-the-vr ! Yeah, it does reduce huge network waterfall.

File: .yarnrc.yml (hunk 1 above)
Comment (SaptarshiSarkar12) [reply .yarnrc.yml:1]: @EmmaDawsonDev @at-the-vr Please let me know if the file is actually related to the error or not.

File: yarn.lock
Comment (SaptarshiSarkar12) [reply yarn.lock:1]: According to https://stackoverflow.com/a/39992365/19187952, committing the yarn.lock file seems necessary. If you (@at-the-vr and @EmmaDawsonDev) have any suggestions to remove the file, then I shall remove them

Comment (SaptarshiSarkar12) [reply yarn.lock:1]: Let me know your opinion @EmmaDawsonDev @at-the-vr

**Comment by SaptarshiSarkar12** [comment]
> Please revert the commit and then only recommit changes to the relevant file and not the yarn files.
> 
> Also, is there a reason why the whole image url needs changing?

@EmmaDawsonDev Please check my comments in the reviews made by @at-the-vr . If you want to remove those files from my commit, then I shall proceed.

Yeah @EmmaDawsonDev , the whole url needs to be changed as the _size_ parameter does not work with the previous github profile picture url.

**Comment by EmmaDawsonDev** [comment]
You did not add any packages so you do not need to commit the yarn files. Please only commit the changes to the maintainers data, thanks!

**Comment by SaptarshiSarkar12** [comment]
> You did not add any packages so you do not need to commit the yarn files. Please only commit the changes to the maintainers data, thanks!

Ok, deleting the yarn lock and yarnrc yml files. Thank you @EmmaDawsonDev!

**Comment by SaptarshiSarkar12** [comment]
@EmmaDawsonDev I have reverted the commits regarding **yarn.lock** and **.yarnrc.yml**. Please re-review this PR.

**Comment by at-the-vr** [comment]
> Please revert the commit and then only recommit changes to the relevant file and not the yarn files.
> 
> Also, is there a reason why the whole image url needs changing?

Yeah its interesting that specifying `https://github.com/emmadawsondev.png | width=150` in maintainers.ts results in a 406 Error because its a redirect to avatar.githubuser route, but with `avatar.githubuser. . . .` route you have the s parameter available for resizing.

**Review by at-the-vr — APPROVED** [review]
completely valid, solid workaround 👍

**Comment by SaptarshiSarkar12** [comment]
> > Please revert the commit and then only recommit changes to the relevant file and not the yarn files.
> > Also, is there a reason why the whole image url needs changing?
> 
> Yeah its interesting that specifying `https://github.com/emmadawsondev.png | width=150` in maintainers.ts results in a 406 Error because its a redirect to avatar.githubuser route, but with `avatar.githubuser. . . .` route you have the s parameter available for resizing.

Yes, you are right @at-the-vr !

**Comment by SaptarshiSarkar12** [comment]
> completely valid, solid workaround :+1:

Thank you for your review @at-the-vr :grin: !

**Comment by SaptarshiSarkar12** [comment]
@EmmaDawsonDev Please review this PR.

**Comment by SaptarshiSarkar12** [comment]
> @EmmaDawsonDev Please review this PR.

@EmmaDawsonDev

**Review by EmmaDawsonDev — APPROVED** [review]
Looks great, thanks for your help!

**Comment by SaptarshiSarkar12** [comment]
> Looks great, thanks for your help!

Thank you for the merge @EmmaDawsonDev :grinning: !

**Pull request merged**